modules/.submodules.json

```diff
@@ -12,14 +12,16 @@
     "netIdSystem",
     "identityLinkIdSystem",
     "sharedIdSystem",
-    "intentIqIdSystem"
+    "intentIqIdSystem",
+    "haloIdSystem"
   ],
   "adpod": [
     "freeWheelAdserverVideo",
     "dfpAdServerVideo"
   ],
   "rtdModule": [
     "browsiRtdProvider",
+    "audigentRtdProvider",
```

Comment:
i still don't fully understand why this is part of the userid module, especially if there's no immediate need for the two modules to communicate.  should this at least be in a separate PR to make it clear?

Reply:
they will be communicating in a forthcoming PR